### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Dangelo-JAN**
# In this milestone I:

Did all the app's style

Note: Kindly check the project requirements
!st. https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md
2nd. https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md
3rd. https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md
4th. https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,4 +1,305 @@
...
+  height: 15vh;
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: #fff;
+  border: none;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+  letter-spacing: 0.5px;
+  text-align: center;
+}
```
Comment (mmsesay) [inline src/App.css]: - [x] Please reduce the height for the form element (title, author, button)

Your design |
:--------------:|
a1

Figma Design |
:-----------------:|
a2

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Dangelo-JAN , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (Dangelo-JAN) [reply src/App.css]: done

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**